root_github_repository.tf

```diff
@@ -0,0 +1,539 @@
+module "configuration" {
+  source  = "./da-terraform-configurations"
+  project = "tdr"
+}
+
+locals {
+  account_secrets = {
+    for environment, _ in module.configuration.account_numbers : environment => {
+      "TDR_${upper(environment)}_ACCOUNT_NUMBER"        = module.configuration.account_numbers[environment]
+      "TDR_${upper(environment)}_TERRAFORM_ROLE"        = module.configuration.terraform_config[environment]["terraform_role"]
+      "TDR_${upper(environment)}_CUSTODIAN_ROLE"        = module.configuration.terraform_config[environment]["custodian_role"]
+      "TDR_${upper(environment)}_STATE_BUCKET"          = module.configuration.terraform_config[environment]["state_bucket"]
+      "TDR_${upper(environment)}_DYNAMO_TABLE"          = module.configuration.terraform_config[environment]["dynamo_table"]
+      "TDR_${upper(environment)}_TERRAFORM_EXTERNAL_ID" = module.configuration.terraform_config[environment]["terraform_external_id"]
+    }
+  }
+}
+
+#module "github_transfer_frontend_repository" {
```

Comment:
Could you delete the commented out code. It just makes it easier to follow as you add more repositories to this repo.

Reply:
yeah, right, deleted them, thanks